### DISCUSSION TITLE
Support specify observable tags/message in TheHive alerter

### DISCUSSION THREAD

**Discussion by pandvan** [jertel/elastalert2#737] [Q&A] [answered]
Would it be possibile to enhance TheHive alerter in order to allow specify in the `hive_observable_data_mapping` (or another key) tags and message fields for an observable? This will allow us to bring more information within an observable.
These fields are supported both in TheHive 3.x and 4.x (see [3.x](https://docs.thehive-project.org/thehive/legacy/thehive3/api/artifact/), [4.x](https://docs.thehive-project.org/thehive/api/observable/create/)).
At the moment the alerter [hardcoded](https://github.com/jertel/elastalert2/blob/master/elastalert/alerters/thehive.py#L39) both fields, respectively to an empty array and `None` object.

**Comment by nsano-rururu** [comment, marked as answer]
There are no plans to support it.
Pull requests are welcome.

**Reply by nsano-rururu** [reply]
It seems that you are maintaining Hive Alerter with custom Elastalert Alerter. It seems that it is supposed to be used in combination with yelp / elastalert. It seems that they are using thehive4py as a library.
https://github.com/Nclose-ZA/elastalert_hive_alerter

**Reply by pandvan** [reply]
If I properly understand, they also create alert artifacts with only dataType and data keys set ([here the code](https://github.com/Nclose-ZA/elastalert_hive_alerter/blob/master/elastalert_hive_alerter/hive_alerter.py#L37))

**Reply by nsano-rururu** [reply]
I think you should use the one that you can achieve.

**Reply by nsano-rururu** [reply]
Nclose-ZA / elastalert_hive_alerter uses TheHive4py, so you can probably do that.
https://github.com/TheHive-Project/TheHive4py

**Reply by luffynextgen** [reply]
I have open this pull request https://github.com/jertel/elastalert2/pulls about that improvement.
